_mobile/android.md

```diff
@@ -273,6 +276,34 @@ packagingOptions {
 }
 ```
 
+Also we have to add all transitive dependencies of our aars. As `pytorch_android` depends on `com.android.support:appcompat-v7:28.0.0` and `com.facebook.soloader:nativeloader:0.8.0`, we need to add them. (In case of using maven dependencies they are added automatically from `pom.xml`).
+
+
+## Custom Build
+
+To reduce the size of binaries you can do custom build of pytorch android with only set of operators required by your model.
+This includes two steps: preparing the list of operators from your model, rebuilding pytorch android with specified list.
+
+1\. Preparation of the list of operators.
+
+List of operators of your serialized torchscript model can be prepared in yaml format using python api function `torch.jit.export_opnames()`:
+```
+import torch, yaml
+m = torch.jit.load("example.pt")
+ops = torch.jit.export_opnames(m)
+f = open('test.yaml', 'w')
+yaml.dump(ops, f)
+```
+2\. Building pytorch android with prepared operators list.
+
+To build pytorch android with prepared yaml list of operators you specify it in environment variable `SELECTED_OP_LIST` and specify as arguments of android ABIs for which you want to build it, by default it builds all 4 android ABIs.
+
+```
+SELECTED_OP_LIST=test.yaml sh scripts/build_pytorch_android.sh x86
+```
+
+After successful build you can integrate the result aar files to your android gradle project, following the steps from previous section of this tutorial (Building PyTorch Android from Source).
+
 ## More Details
```

Comment:
Rather than "More Details", should we change this header to something more descriptive? Suggestion - "Javadoc" or "PyTorch Android API"